blockchains/cardano/cardano_worker.js

```diff
@@ -191,7 +194,6 @@ class CardanoWorker extends BaseWorker {
     else {
       transactions = await this.getTransactions(fromBlock, this.lastConfirmedBlock);
       if (transactions.length === 0) {
-        this.lastExportedBlock = this.lastConfirmedBlock;
```

Comment:
That's one potential place where can break the `this.lastExporedBlock` logic. However we shouldn't reach here if `this.lastConfirmedBlock < this.lastExportedBlock`. Anyways removing this as it is unsafe.